src/tools/pub/pubspec.md

```diff
@@ -398,6 +403,43 @@ the pattern is considered relative to the package's root directory.
   Support for the `false_secrets` field was added in Dart 2.15.
 {{site.alert.end}}
 
+### Screenshots
+
+Packages may showcase their widgets or other visual elements using screenshots.
+
+A package can include up to 10 screenshots. 
+These are specified in the pubspec as a list under the `screenshots` field. 
+Each screenshot is specified by a `description` and a `path`. 
+The `description` is a string of at most 160 characters
+describing the screenshot. 
+For example:
+
+```yaml
+screenshots:
+  - description: 'description of screenshot'
+    path: path/to/image/in/package/500x500.webp
+  - description: 'description of second screenshot'
+    path: path/to/image/in/package.png
+```
+
```

Comment:
**suggestion:** Change the following specifications to a bulleted list.

Example:
```
Screenshots must meet the following requirements:
- Use one of the following file types: `png`, `jpg`, `gif`, and `webp`. 
- Use a static or animated image.
- Keep to a file size of less than 4 MB each.
- Use a static, square image for your first screenshot. 
  Pub.dev generates the package's thumbnail image from the first screenshot.
  - If this screenshot uses animation, pub.dev uses its first frame.
  - If this screenshot is not square, pub.dev (stretches|trims|pads) the thumbnail.
```